webapp/src/pages/Settings.tsx

```diff
@@ -637,55 +647,83 @@ const Settings: React.FC = () => {
-          [`& .fixedWidthInput .${inputClasses.root}`]: { width: "12ch" },
-          [`& .${inputClasses.input}`]: { fontSize: 14, padding: 0 },
-          [`& .${inputLabelClasses.root}`]: { fontWeight: "bold" },
-        }}
-      >
-        <Typography variant="subtitle1" marginBottom={3}>
-          View and edit certain fields from your config file. Once your changes
-          are saved, expect some delays for recomputing the affected tasks.
-        </Typography>
-        <AccordionLayout
-          name="Project Configuration"
-          description="View the fields that define the dataset to load in Azimuth."
-          link="reference/configuration/project/"
-        >
-          {getProjectConfigSection()}
-        </AccordionLayout>
-        <AccordionLayout
-          name="Model Contract Configuration"
-          description="View and edit some fields that define the ML pipelines and the metrics."
-          link="reference/configuration/model_contract/"
-        >
-          {getModelContractConfigSection()}
-        </AccordionLayout>
-        <AccordionLayout
-          name="Analyses Customization"
-          description="Enable or disable some analyses and edit corresponding thresholds."
-          link="reference/configuration/analyses/"
+    <>
+      <DialogTitle id="config-dialog-title">
+        <Box display="flex" justifyContent="space-between">
+          <Typography variant="subtitle1">
+            View and edit certain fields from your config file. Once your
+            changes are saved, expect some delays for recomputing the affected
+            tasks.
+          </Typography>
+          <Tooltip title="close" placement="bottom">
+            <IconButton
+              size="small"
+              color="primary"
+              sx={{ padding: 0 }}
+              onClick={() => {
+                if (isUpdatingConfig) {
+                  window.alert(CONFIG_UPDATE_MESSAGE);
+                } else if (
+                  Object.keys(partialConfig).length === 0 ||
+                  window.confirm(
+                    "Are you sure you want to discard all your changes?"
+                  )
+                ) {
+                  setOpen(false);
+                }
+              }}
+            >
+              <Close />
+            </IconButton>
+          </Tooltip>
+        </Box>
+      </DialogTitle>
+      <DialogContent dividers>
+        <Box
+          sx={{
+            flex: 1,
+            padding: 2,
+            overflow: "auto",
+            [`& .${formControlLabelClasses.labelPlacementStart}`]: {
+              justifyContent: "flex-end",
+              marginLeft: 0,
+            },
+            [`& .${formGroupClasses.root}`]: { marginX: 2, marginBottom: 2 },
+            [`& .fixedWidthInput .${inputClasses.root}`]: { width: "12ch" },
+            [`& .${inputClasses.input}`]: { fontSize: 14, padding: 0 },
+            [`& .${inputLabelClasses.root}`]: { fontWeight: "bold" },
+          }}
```

Comment:
A couple of things:
- The `flex: 1` has no effect since its parent `DialogContent` is a `display: block`.
- The `overflow: "auto"` is already handled by the parent `DialogContent`.
- The `DialogContent` already has some padding, which I find enough.

So, I think we can remove that `Box` and move the `sx` prop to the `DialogContent`. Does that make sense?
```suggestion
      <DialogContent
        dividers
        sx={{
          [`& .${formControlLabelClasses.labelPlacementStart}`]: {
            justifyContent: "flex-end",
            marginLeft: 0,
          },
          [`& .${formGroupClasses.root}`]: { marginX: 2, marginBottom: 2 },
          [`& .fixedWidthInput .${inputClasses.root}`]: { width: "12ch" },
          [`& .${inputClasses.input}`]: { fontSize: 14, padding: 0 },
          [`& .${inputLabelClasses.root}`]: { fontWeight: "bold" },
        }}
```

Reply:
Right!.. Thanks :)